.tekton/tasks/task-deploy-to-code-engine.yml

```diff
@@ -0,0 +1,209 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
We might want to consider publishing this to the tekton-catalog repo. It's useful to have for everyone.